rust/README.md

```diff
@@ -6,37 +6,32 @@ Earthly's official collection of Rust [functions](https://docs.earthly.dev/docs/
 
 First, import the library up in your Earthfile:
 ```earthfile
-VERSION --global-cache 0.7
+VERSION 0.8
 IMPORT github.com/earthly/lib/rust:<version/commit> AS rust
 ```
-> :warning: Due to [this issue](https://github.com/earthly/earthly/issues/3490), make sure to enable `--global-cache` in the calling Earthfile, as shown above.
+> *Due to [this issue](https://github.com/earthly/earthly/issues/3490), make sure to enable `--global-cache` in the calling Earthfile, as shown above.*
```

Comment:
This is no longer shown above, probably because it's not needed with 0.8 any more?